docs/src/interop.md

```diff
@@ -8,7 +8,7 @@ The documentaiton below provides some essential functionality, such as creating
 
 For a detailed overview, or if you want to know how AtomsBase.jl operates behind the scenes, refer to its official documentation.
 
-## Creating AtomsBase Systems
+#### Creating AtomsBase Systems
```

Comment:
Why not have this be 3 and `AtomsBase.jl` be 2?  The header level should generally reflect the text hierarchy, and the size of the rendered text is a styling side effect.  If we don't like the relative sizes of the doc theme then we can try changing that.

It looks like all the remaining headers should be changed to 3 and `Fermi.jl` left at 2 since it's non-AtomsBase.